perf/roles/trex/defaults/main.yml

```diff
@@ -2,5 +2,8 @@
 
 trex_repo: https://github.com/rh-nfv-int/trex-core
 trex_tag: rhosp-playbooks
-trex_dir: /tmp/trex-core
 trex_opts: ''
+trex_release_url: "https://trex-tgn.cisco.com/trex/release/latest"
+trex_release_filename: "trex_latest.tar.gz"
+trex_install_dir: "/opt/trex"
+trex_version: 'v3.02'
```

Comment:
duplicate?

Reply:
Actually I only need Trex Installation Path in trex-ndr role. So, now I  kept only one variable : `trex_install_dir`